spec/adapters/googlecharts_spec.rb

```diff
@@ -88,4 +100,52 @@
       expect{Daru::View::Table.new("daru")}.to raise_error(ArgumentError) 
     end
   end
+
+  describe "#generate_body" do
+    it "generates valid JS of the google chart" do
+      js = area_chart_chart.adapter.generate_body(area_chart_chart.chart)
+      expect(js).to match(/<script>/i)
+      expect(js).to match(/google.visualization.DataTable\(\);/i)
+      expect(js).to match(
+        /data_table.addColumn\(\{\"type\":\"string\",\"label\":\"Year\"\}\);/i)
+      expect(js).to match(
+        /data_table.addRow\(\[\{v: \"2013\"\}\]\);/i)
+      expect(js).to match(/google.visualization.AreaChart/i)
+      expect(js).to match(/chart.draw\(data_table, \{\}/i)
+    end
+    it "generates valid JS of the google data_table" do
+      js = area_chart_table.adapter.generate_body(area_chart_table.table)
+      expect(js).to match(/<script>/i)
+      expect(js).to match(/google.visualization.DataTable\(\);/i)
+      expect(js).to match(
+        /data_table.addColumn\(\{\"type\":\"string\",\"label\":\"Year\"\}\);/i)
+      expect(js).to match(
+        /data_table.addRow\(\[\{v: \"2013\"\}\]\);/i)
+    end
+  end
+
+  describe "#generate_html" do
+    it "generates valid html of the google chart" do
+      js = area_chart_chart.adapter.generate_html(area_chart_chart.chart)
+      expect(js).to match(/html/i)
+      expect(js).to match(/<script>/i)
+      expect(js).to match(/google.visualization.DataTable\(\);/i)
+      expect(js).to match(
+        /data_table.addColumn\(\{\"type\":\"string\",\"label\":\"Year\"\}\);/i)
+      expect(js).to match(
+        /data_table.addRow\(\[\{v: \"2013\"\}\]\);/i)
+      expect(js).to match(/google.visualization.AreaChart/i)
+      expect(js).to match(/chart.draw\(data_table, \{\}/i)
+    end
+    it "generates valid html of the google data_table" do
+      js = area_chart_table.adapter.generate_html(area_chart_table.table)
+      expect(js).to match(/html/i)
+      expect(js).to match(/<script>/i)
+      expect(js).to match(/google.visualization.DataTable\(\);/i)
+      expect(js).to match(
+        /data_table.addColumn\(\{\"type\":\"string\",\"label\":\"Year\"\}\);/i)
+      expect(js).to match(
+        /data_table.addRow\(\[\{v: \"2013\"\}\]\);/i)
+    end
+  end
```

Comment:
expecting similar test cases for `init_script`, `export_html_file`,  try for `init_iruby`. Also please test with few other chart types as well.